web/source/osk/anchoredOskView.ts

```diff
@@ -150,17 +150,27 @@ namespace com.keyman.osk {
         return keymanweb['getOskHeight']();
       }
 
-      var oskHeightLandscapeView=Math.floor(Math.min(screen.availHeight,screen.availWidth)/2),
+      let baseWidth  = document?.documentElement?.clientWidth;
```

Comment:
Could you add a comment explaining the rationale for the measurement source choices for future maintainers?